lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -80,12 +80,39 @@ def get_min_grade(self, req):
         return req['criteria']['min_grade'] * 100
 
 
+class CreditCourseRequirementsSerializer(serializers.Serializer):
+    """
+    Serializer for credit_course_requirements
+    """
+    dashboard_url = serializers.SerializerMethodField()
+    eligibility_status = serializers.CharField()
+    requirements = RequirementSerializer(many=True)
+
+    def get_dashboard_url(self, creditCourseRequirement):
```

Comment:
So when you have a parameter that you don't use, convention is to put an underscore there to indicate this is an input we aren't using at all.
```
def get_dashboard_url(self, _):